md/codeBlock/fenced.ts

```diff
@@ -39,7 +39,16 @@ export function create(
   return fence + _infoString + "\n" + code + "\n" + fence;
 }
 
-export function parse(codeBlock: string) {
+export function parse(
+  codeBlock: string,
+): {
+  type: "fenced";
+  char: FenceChar;
+  fence: string;
+  lang: string | undefined;
+  meta: string | undefined;
+  code: string;
+} {
```

Comment:
This really needs to be exported. It has the shape of an AST node and might want to be named accordingly.

Reply:
Fixed in #50